py/server/deephaven/updateby.py

```diff
@@ -944,10 +1173,10 @@ def rolling_min_time(ts_col: str, cols: Union[str, List[str]], rev_time: Union[i
             current row timestamp (inclusive), this is a purely backwards looking window
         |  `rev_time = "-PT00:05:00", fwd_time = "PT00:10:00"` - contains rows from 5m following through 10m
             following the current row timestamp (inclusive), this is a purely forwards looking window
-    
+
     Args:
         ts_col (str): the timestamp column for determining the window
-        cols (Union[str, List[str]]): the column(s) to be operated on, can include expressions to rename the output,
+        cols (Union[str, list[str]]): the column(s) to be operated on, can include expressions to rename the output,
             i.e. "new_col = col"; when empty, update_by performs the rolling minimum operation on all columns.
         rev_time (int): the look-behind window size, can be expressed as an integer in nanoseconds or a time
```

Comment:
More wrong types.  You need to point AI at this entire PR and have it identify and correct all of the pydoc problems I keep commenting on.  They are all over the place.  I certainly will not find every one.